components/widgets/admin_console/formatted_admin_header.test.tsx

```diff
@@ -51,36 +51,4 @@ describe('components/widgets/admin_console/FormattedAdminHeader', () => {
 `
         );
     });
-
```

Comment:
any reason why the test was removed?

Reply:
There's no longer a reason reason to check "mandatory" props with typescript, actually, here we were testing something that is not "part of our code".

Reply:
👍